README.md

```diff
@@ -56,6 +56,35 @@ cur.execute('SELECT * FROM system.runtime.nodes')
 rows = cur.fetchall()
 ```
 
+# Oauth Authentication
+To enable GCS access, Oauth authentication support is added by passing in a `shadow.json` file of a service account.
+Following example shows a use case where both Kerberos and Oauth authentication are enabled.
+
+```python
+import getpass
+import prestodb
+from prestodb.client import PrestoRequest, PrestoQuery
+from requests_kerberos import DISABLED
+
+kerberos_auth = prestodb.auth.KerberosAuthentication(
+   mutual_authentication=DISABLED,
+   service_name='kerberos service name',
+   force_preemptive=True,
+   hostname_override='example.com')
+
+req = PrestoRequest(
+        host='GCP coordinator url',
+        port=443,
+        user=getpass.getuser(),
+        service_account_file='Service account json file path',
+        http_scheme='https',
+        auth=kerberos_auth
+    )
```

Comment:
Seems to be an extra level of indentation for line 76-82

Reply:
good catch, thanks!